app/src/main/res/layout-xlarge/activity_login.xml

```diff
@@ -121,7 +121,7 @@
                         android:layout_marginStart="@dimen/standard_gap"
                         app:passwordToggleEnabled="true">
 
-                        <android.support.design.widget.TextInputEditText
+                        <com.google.android.material.textfield.TextInputEditText
```

Comment:
Shouldn't this be replaced in all the activity_login.xml (landscape and the usual one) ?

Reply:
And also all the "android.support.design.widget.TextInputEditText" should be replaced by "com.google.android.material.textfield.TextInputEditText" ?

Reply:
for activity_login.xml (regular) the change is already present :)
```
<com.google.android.material.textfield.TextInputEditText
                        android:id="@+id/login_username"
                        android:layout_width="match_parent"
                        android:layout_height="wrap_content"
                        android:hint="@string/username"
                        android:imeOptions="flagNoExtractUi"
                        android:inputType="textNoSuggestions">

                        <requestFocus />

                    </com.google.android.material.textfield.TextInputEditText>
```
```
<com.google.android.material.textfield.TextInputEditText
                        android:id="@+id/login_password"
                        android:layout_width="match_parent"
                        android:layout_height="wrap_content"
                        android:hint="@string/password"
                        android:imeOptions="flagNoExtractUi"
                        android:inputType="textPassword" />
```
```
<com.google.android.material.textfield.TextInputEditText
                        android:id="@+id/login_two_factor"
                        android:layout_width="match_parent"
                        android:layout_height="wrap_content"
                        android:hint="@string/_2fa_code"
                        android:imeOptions="flagNoExtractUi"
                        android:inputType="textNoSuggestions"
                        android:visibility="gone"
                        tools:visibility="visible" />
```
Similarly for activity_login.xml (land)
```
<com.google.android.material.textfield.TextInputEditText
                        android:id="@+id/login_username"
                        android:layout_width="match_parent"
                        android:layout_height="wrap_content"
                        android:hint="@string/username"
                        android:imeOptions="flagNoExtractUi"
                        android:inputType="textNoSuggestions">

                        <requestFocus />

                    </com.google.android.material.textfield.TextInputEditText>
```
```
<com.google.android.material.textfield.TextInputEditText
                        android:id="@+id/login_password"
                        android:layout_width="match_parent"
                        android:layout_height="wrap_content"
                        android:hint="@string/password"
                        android:imeOptions="flagNoExtractUi"
                        android:inputType="textPassword" />
```
```
<com.google.android.material.textfield.TextInputEditText
                        android:id="@+id/login_two_factor"
                        android:layout_width="match_parent"
                        android:layout_height="wrap_content"
                        android:hint="@string/_2fa_code"
                        android:imeOptions="flagNoExtractUi"
                        android:inputType="textNoSuggestions"
                        android:visibility="gone"
                        tools:visibility="visible" />
```
So only change needed was in the activity_login.xml (xlarge) where it was crashing :)

Reply:
@ashishkumar0207 I am not sure about replacing **"android.support.design.widget.TextInputEditText"** with **"com.google.android.material.textfield.TextInputEditText"** because it was'nt the reason for app crash, but I can make the change if you would like me to do :)

Reply:
> for activity_login.xml (regular) the change is already present :)
> 
> ```
> <com.google.android.material.textfield.TextInputEditText
>                         android:id="@+id/login_username"
>                         android:layout_width="match_parent"
>                         android:layout_height="wrap_content"
>                         android:hint="@string/username"
>                         android:imeOptions="flagNoExtractUi"
>                         android:inputType="textNoSuggestions">
> 
>                         <requestFocus />
> 
>                     </com.google.android.material.textfield.TextInputEditText>
> ```
> ```
> <com.google.android.material.textfield.TextInputEditText
>                         android:id="@+id/login_password"
>                         android:layout_width="match_parent"
>                         android:layout_height="wrap_content"
>                         android:hint="@string/password"
>                         android:imeOptions="flagNoExtractUi"
>                         android:inputType="textPassword" />
> ```
> ```
> <com.google.android.material.textfield.TextInputEditText
>                         android:id="@+id/login_two_factor"
>                         android:layout_width="match_parent"
>                         android:layout_height="wrap_content"
>                         android:hint="@string/_2fa_code"
>                         android:imeOptions="flagNoExtractUi"
>                         android:inputType="textNoSuggestions"
>                         android:visibility="gone"
>                         tools:visibility="visible" />
> ```
> Similarly for activity_login.xml (land)
> 
> ```
> <com.google.android.material.textfield.TextInputEditText
>                         android:id="@+id/login_username"
>                         android:layout_width="match_parent"
>                         android:layout_height="wrap_content"
>                         android:hint="@string/username"
>                         android:imeOptions="flagNoExtractUi"
>                         android:inputType="textNoSuggestions">
> 
>                         <requestFocus />
> 
>                     </com.google.android.material.textfield.TextInputEditText>
> ```
> ```
> <com.google.android.material.textfield.TextInputEditText
>                         android:id="@+id/login_password"
>                         android:layout_width="match_parent"
>                         android:layout_height="wrap_content"
>                         android:hint="@string/password"
>                         android:imeOptions="flagNoExtractUi"
>                         android:inputType="textPassword" />
> ```
> ```
> <com.google.android.material.textfield.TextInputEditText
>                         android:id="@+id/login_two_factor"
>                         android:layout_width="match_parent"
>                         android:layout_height="wrap_content"
>                         android:hint="@string/_2fa_code"
>                         android:imeOptions="flagNoExtractUi"
>                         android:inputType="textNoSuggestions"
>                         android:visibility="gone"
>                         tools:visibility="visible" />
> ```
> So only change needed was in the activity_login.xml (xlarge) where it was crashing :)

Yea, seems like I was a few commits away on master, my bad

Reply:
> @ashishkumar0207 I am not sure about replacing **android.support.design.widget.TextInputEditText** with **com.google.android.material.textfield.TextInputEditText** because it wasn't the reason for app crash, but I can make the change if you would like me to do :)

If that is the case, IMO, lets not do this, in this PR.